bcda/auth/middleware_test.go

```diff
@@ -80,7 +81,34 @@ func (s *MiddlewareTestSuite) TestReturn400WhenInvalidTokenAuthWithInvalidSignat
 	assert.Nil(s.T(), err)
 }
 
-//unit test
+// integration test: makes HTTP request & asserts HTTP response
+func (s *MiddlewareTestSuite) TestReturn401WhenExpiredToken() {
+	client := s.server.Client()
+	expiredToken := jwt.NewWithClaims(jwt.SigningMethodRS512, &auth.CommonClaims{
+		StandardClaims: jwt.StandardClaims{
+			Issuer:    "ssas",
+			ExpiresAt: time.Now().Unix(),
+		},
+		ClientID: uuid.New(),
+		SystemID: uuid.New(),
+		Data:     `{"cms_ids":["A9994"]}`,
+	})
+
+	req, err := http.NewRequest("GET", fmt.Sprintf(constants.ServerPath, s.server.URL), nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	req.Header.Add("Authorization", fmt.Sprintf(bearerStringMsg, expiredToken))
+	resp, err := client.Do(req)
+
+	assert.NotNil(s.T(), resp)
```

Comment:
This looks great! I'm not sure I see it here- are we also testing for this acceptance criteria in the ticket? 

`the detailsCode in FHIR Operational Outcome (response body) is "Expired Token"`

Reply:
@laurenkrugen-navapbc yes!

Reply:
sorry - I think my eyes are missing it; I see the assertion that the resp is not not nil and that we are getting the correct status code, but I don't see a comparison for the response body.. can you point me to the line or give me a quick walkthrough?

Reply:
@laurenkrugen-navapbc up to date!

Reply:
Thank you for catching this :)